packages/medusa/src/loaders/module-definitions/index.ts

```diff
@@ -0,0 +1,34 @@
+import glob from "glob"
+import path from "path"
+import resolveCwd from "resolve-cwd"
+
+import { ModuleDefinition, ModuleResolution } from "../../types/global"
+
+export default (configModule) => {
```

Comment:
suggestion: should we type the configModule here with the minimum expected ` { modules: ... }